portfolio-edit/Panel.php

```diff
@@ -0,0 +1,76 @@
+<?php
+$db = new PDO('mysql:host=192.168.20.20;dbname=Project_2', 'root', '');
+$db-> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$sql = ' SELECT `name`, `url` FROM `portfolio`;';
+$query = $db->query($sql);
+$result = $query->fetchAll();
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+<body>
+
+<h1> Admin page </h1>
+<table>
+    <thead>
+        <tr>
+            <th> Projects </th>
+            <th> URL </th>
+            <th> Edit </th>
+            <th> Delete</th>
+        </tr>
+    </thead>
+    <tbody>
+    <?php
+    foreach($result as $project){
```

Comment:
If you rename $result to $projects - this loop will make more sense
foreach($projects as $project)...